### PR TITLE
Fix #3001: Add native html title for all map buttons

### LINKED ISSUE #3001: 3D: Tooltip on nav btns and help (ctrl + right or left click)
The help is nice, but the user might want something more direct than a big popup window.

A tooltip on both 3D nav buttons as contextual help would be nice. (Desktop only)

The ctrl + right or left click for navigating in 3D is obvious to us but not to all of our users.
It should be more obvious you can use the keyboard to navigate.
This could show up when hovering on the 3D buttons as well.
(I know it's platform dependent but we can detect it and adapt the message accordingly)

@davidoesch What do you think?

translation

tilt
[DE] neigen
[FR] incliner
[IT]  inclina
[EN] tilt
[RM]  inclinar

Rotate
[DE] drehen
[FR] tourner
[IT]  ruotare
[EN] rotate
[RM]  volver

### PR DESCRIPTION
**Pull request by oterral**
Fix https://github.com/geoadmin/mf-geoadmin3/issues/3001

[Test](https://mf-geoadmin3.int.bgdi.ch/teo_tooltips/index.html?lang=en)

This PR add title for all map buttons : 
- Geolocation
- Zoom in
- Zoom out
- 3d
- Time
- Rotate
- Background selector
- Offline
-  3d controls (tilt, rotate)

### CODE REVIEW

**Comment by gjn** [comment]
Set to WIP until we have all translations.

**Comment by oterral** [comment]
All translations available

**Comment by gjn** [comment]
Nice improvement. Thanks alot.

File: src/locales/fr.json [hunk 1]
```diff
@@ -380,10 +387,19 @@
 "test_host_warning": "TESTSITE – DO NOT-SHARE -  NOT FOR OPERATIONAL USE<br/>This site is for testing purposes only. It's not meant for operational use and there's no guarantee whatsoever.",
 "text_to_display": "Description du lien",
 "tile": "Feuille",
+"tilt3d_active": "Passer en 2d",
+"tilt3d_disabled": "Vous ne pouvez pas utiliserce fond de plan en 3d",
```
Comment (loicgasser) [inline src/locales/fr.json]: typo!!!!

**Pull request merged**